tests/unit/dojson/test_dojson_hep_bd1xx.py

```diff
@@ -970,42 +828,66 @@ def test_authors_from_100__a_v_w_x_y_and_100_a_v_w_y():
     )
 
     expected = [
-        {
+        {  # First that record due to the calling order of the dojson rule (both are 100 field)
```

Comment:
Here you're cheating. You're modifying the test (which is correct) instead of the code (which is incorrect).